server/cli/range.go

```diff
@@ -53,22 +51,20 @@ func runLsRanges(cmd *cobra.Command, args []string) {
 		startKey = engine.KeyMeta2Prefix
 	}
 
-	kv, err := makeKVClient()
-	if err != nil {
-		fmt.Fprintf(osStderr, "failed to initialize KV client: %s", err)
-		osExit(1)
+	kv := makeDBClient()
```

Comment:
I'd actually like `client.DB` to be renamed to `client.KV` and for the existing `client.KV` to become something else. But for now I'll rename to `kvDB`.